workitem_test.go

```diff
@@ -104,9 +106,10 @@ func TestCreateWI(t *testing.T) {
 	payload := app.CreateWorkitemPayload{
 		Type: "system.bug",
 		Fields: map[string]interface{}{
-			"system.title":   "Test WI",
-			"system.creator": "tmaeder",
-			"system.state":   "new",
+			"system.title":          "Test WI",
+			"system.creator":        "tmaeder",
+			"system.state":          "new",
+			"system.remote_item_id": "http://github.com/sbose78/project/1",
```

Comment:
Work item can be non-remote also, right? So, do we really need to modify this test?